### PR TITLE
chore: migrate webui launch notebook api [DET-4821]

### PR DESCRIPTION
**Pull request by apizzini**
## Test Plan

I tested this by triggering "launch notebook" and "launch cpu-only notebook".

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: proto/src/determined/api/v1/notebook.proto [hunk 1]
```diff
@@ -95,6 +96,9 @@ message LaunchNotebookRequest {
 }
 // Response to LaunchNotebookRequest.
 message LaunchNotebookResponse {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "notebook" ] }
```
Comment (apizzini) [inline proto/src/determined/api/v1/notebook.proto:100]: @stoksc can you pls check if setting `notebook` as required here is acceptable?

**Review by hamidzr — APPROVED** [review]
LGTM

File: webui/react/src/services/decoder.ts [hunk 2]
```diff
@@ -152,6 +132,7 @@ export const mapV1NotebookToCommandTask = (notebook: Sdk.V1Notebook): types.Comm
     id: notebook.id,
     name: notebook.description,
     resourcePool: notebook.resourcePool,
```
[the remark this answers is not in the document]
Comment (hamidzr) [reply webui/react/src/services/decoder.ts:134]: also naming wise we updated the style guide today in the sync

[the remark this answers is not in the document]
Comment (apizzini) [reply webui/react/src/services/decoder.ts]: @hamidzr what do you think about this? https://github.com/determined-ai/determined/pull/1943/commits/b1f09ca3ac873a0dceaa85ac8b2bd1e888934a3f

[the remark this answers is not in the document]
Comment (hamidzr) [reply webui/react/src/services/decoder.ts]: looks great.

**Pull request merged**